### ISSUE TITLE
[Question] Why development stalled ?

### ISSUE THREAD

**Issue by kirantpatil** [bonsaiERP/bonsaiERP#10]
Hi,

What is the reason development has stalled ?

Anyone maintains it or abandoned ?

If we develop any new modules on top it, will there be any community help available ?

Any other Rails ERP projects which are maintained ?

Thanks.

**Comment by boriscy** [comment]
I have worked on this project for a long time alone in hope that some time I will make money of it, I think I have made many errors especially not creating a community but I can't work on this anymore, you can fork it and work if you want.

**Comment by kirantpatil** [comment]
Thanks. We will see if we can use it to develop further. 

We need to develop Invoice, Payment, Receipt, Purchase order and Reports.

We may develop and contribute back, will your help is available since our developers are Junior programmers ?

Thanks.

**Comment by kirantpatil** [comment]
Hi @boriscy,

How stable is this code ?

Was it is production for some time ?

We need ERP solution can we still rely on it ? 

If yes then we would like to port it to RAILS 7. What are your thoughts ?

Thanks.

**Comment by boriscy** [comment]
Yes the Code was stable It was made to work as a service but nevera took
off, It would be great that you migrate to rails 7.
